tests/models/reservoir_evaporation_areafromfile.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Add a function to `tests/test_run.py` to load, run and check the output of this example.

Reply:
done!